ldms/man/ldmsd_sampler_discovery.man

```diff
@@ -0,0 +1,245 @@
+\" Manpage for ldmsd_sampler_discovery
+.TH man 7 "27 March 2024" "v5" "LDMSD Sampler Discovery man page"
+
+.\""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""/.
+.SH NAME
+ldmsd_sampler_disconvery - Manual for LDMSD Sampler Discovery
+
+.\""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""/.
+.SH SYNOPSIS
+
+**Sampler side Commands**
+
+.IP \fBadvertise_add
+.RI "name=" NAME " xprt=" XPRT " host=" HOST " port=" PORT
+.RI " [auth=" AUTH_DOMAIN "]"
+
+.IP \fBadvertise_start
+.RI "name=" NAME
+
+.IP \fBadvertise_stop
+.RI "name=" NAME
+
+.IP \fBadvertise_del
+.RI "name=" NAME
+
+.IP \fBadvertise_status
+.RI " [name=" NAME "]"
+
+.PP
+**Aggregator Side Commands**
+
+.IP \fBprdcr_listen-add
```

Comment:
I suspect there is a typo here: s/-/_/

Reply:
@morrone Thanks for catching this and looking into the details!